src/js/containers/ApplyDefaultSettingsCard/ApplyDefaultSettingsCard.js

```diff
@@ -17,26 +16,13 @@ const VALUE = true;
 class ApplyDefaultSettingsCard extends Component {
     constructor(props) {
```

Comment:
We don't need the constructor at all anymore since it only served to create the local state right?